styles/main_styles.css

```diff
@@ -82,477 +84,294 @@ li {
-  height: 204px;
-  left: 1080px;
-  top: 92px;
-}
-
-.light_right {
-  position: absolute;
-  width: 204px;
-  height: 87px;
-  left: 1000px;
-  top: 61px;
-  transform: rotate(-15deg);
-}
-
-.deep_left {
-  position: absolute;
-  width: 70.48px;
-  height: 50.24px;
-  left: 90.2px;
-  top: 60.68px;
-  transform: rotate(5deg);
-  z-index: 100;
-}
-
-.light_left {
-  position: absolute;
-  width: 108.67px;
-  height: 59.44px;
-  left: 35.6px;
-  z-index: 0;
-  top: 28.92px;
-  transform: rotate(-25deg);
-}
-
-.roll_ball {
-  position: absolute;
-  width: 47px;
-  height: 47px;
-  left: 720px;
-  top: 749px;
-}
-
-.roll_bar {
-  position: absolute;
-  width: 161.41px;
-  height: 68.87px;
-  left: 605.44px;
-  top: 750px;
-  transform: rotate(7deg);
-}
-
-.bowl {
-  position: absolute;
-  width: 36.9px;
-  height: 18.45px;
-  left: 250.7px;
-  top: 817.93px;
-  transform: rotate(10deg);
-}
-
-.hook {
-  position: absolute;
-  width: 46px;
-  height: 46px;
-  left: 226.06px;
-  top: 830.37px;
-  transform: rotate(0deg);
-}
-
-.ball {
-  position: absolute;
-  width: 25px;
-  height: 25px;
-  left: 580px;
-  top: 105px;
-}
-
-.bar {
-  position: absolute;
-  width: 56.01px;
-  height: 20.02px;
-  left: 567.25px;
-  top: 87.57px;
-}
-
-/* For mobile */
-@media (max-width: 900px) {
-  .hero-section {
-    margin: 0;
-    border-bottom-left-radius: 25% 45px;
-    height: 718px;
-    width: 100vw;
-    background-color: white;
-  }
+.action {
+  margin-top: 8px;
+  padding: 3px 10px 2px 1px;
+}
 
-  .showcase {
-    margin-top: -80px;
-  }
+.see-project :hover {
+  background-color: blue;
+  color: white;
+}
 
-  .intro {
-    line-height: 35px;
-    font-size: 35px;
-    width: 210px;
-  }
+.see-project {
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  background-color: white;
+  padding: 6px;
+  color: #6070ff;
+  transition: 3ms;
+}
 
-  #menu {
-    display: block;
-    color: #075cfa;
-    float: right;
-    margin-right: 10px;
-    padding: 3px;
-  }
+.about-section {
+  font-family: "poppins", sans-serif;
+  background-color: white;
+  border-top-right-radius: 25% 45px;
+  padding: 70px 24px;
+  display: flex;
+  flex-direction: column;
+}
 
-  .logo {
-    float: left;
-    padding: 3px;
-  }
+.for-space {
+  padding: 20px 1px;
+}
 
-  .social {
-    position: absolute;
-    left: -15px;
-  }
+.main-list {
+  display: block;
+  width: 100%;
+}
 
-  nav a {
-    display: none;
-  }
+.skill-headline {
+  display: grid;
+  grid-template-columns: auto 30px;
+}
 
-  .deep_left {
-    position: absolute;
-    width: 40.48px;
-    height: 20.24px;
-    left: -25px;
-    top: -5px;
-  }
+.skill-headline .icon {
+  margin-top: 22px;
+  width: 20px;
+}
 
-  .light_left {
-    position: absolute;
-    width: 80.67px;
-    height: 29.44px;
-    left: -50px;
-    top: 15px;
-  }
+.skill-item ul {
+  display: grid;
+  grid-template-rows: auto auto auto;
+  font-size: 15px;
+  color: #253858;
+  padding-left: 0.1px;
+}
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements, you are to stick to the design as much as possible. Please  align the items horizontally to match the original design

### Your design
![Abt1](https://user-images.githubusercontent.com/63964646/149413314-da900692-42c8-4d0c-9093-07fe34e3e3fd.PNG)


### Original Design

![Abt2](https://user-images.githubusercontent.com/63964646/149413340-7c9356b2-6799-4fc6-adfd-36a472da581c.PNG)